pkg/agent/openflow/multicluster.go

```diff
@@ -184,3 +186,22 @@ func (f *featureMulticluster) snatConntrackFlows(serviceCIDR net.IPNet, localGat
 	)
 	return flows
 }
+
+func (f *featureMulticluster) l3FwdFlowToPodViaTun(
+	localGatewayMAC net.HardwareAddr,
+	podIP net.IP,
+	tunnelPeer net.IP) binding.Flow {
+	ipProtocol := getIPProtocol(podIP)
+	// This generates the flow to forward cross-cluster request packets based
+	// on Pod IP.
+	return L3ForwardingTable.ofTable.BuildFlow(priorityHigh).
```

Comment:
Same question as above, why it needs to be high priority and override the existing flows with normal priority?

Reply:
Changed to normal priority.